pkg/cmd/pulumi/about.go

```diff
@@ -188,6 +189,16 @@ func getSummaryAbout(ctx context.Context, transitiveDependencies bool, selectedS
 		tmp := getBackendAbout(backend)
 		result.Backend = &tmp
 	}
+
+	if result.CurrentStack != nil && strings.Contains(result.CurrentStack.Name, "/") {
+		// if the stack name contains a /, it's $org/$stack, so we need to insert project
+		parts := strings.Split(result.CurrentStack.Name, "/")
+		result.StackReference = fmt.Sprintf("%v/%v/%v", parts[0], proj.Name, parts[1])
+	} else if result.CurrentStack != nil && result.Backend != nil {
+		// the stack name is $stack with the current user as the org
+		result.StackReference = fmt.Sprintf("%v/%v/%v", result.Backend.User, proj.Name, result.CurrentStack.Name)
```

Comment:
This isn't correct for filestate, also we should probably just ask the StackReference for the qualified name, rather than trying to reconstruct it. 
Something like `FullyQualifiedName() tokens.QName`, which for filestate just returns the same as Name but for httpstate returns the full name regardless of the current defaults for org and project.

Reply:
I still want to implement stack refs for filestate, so for now would it be OK for filestate stacks (`localBackend`) to return nil or `""` while we decide on how to handle the org?

Reply:
I've implemented this and (overdue? or duplicate?) opened #11390 so we can formally decide on this.

For local backends, we just don't render a fully qualified stack name.